Main/StackWalker/StackWalker.cpp

```diff
@@ -1439,14 +1440,34 @@ void StackWalker::OnSymInit(LPCSTR szSearchPath, DWORD symOptions, LPCSTR szUser
     buffer[STACKWALK_MAX_NAMELEN - 1] = 0;
     OnOutput(buffer);
   }
+#elif (_MSC_VER >= 1800) && (_WIN32_WINNT > _WIN32_WINNT_VISTA)

```

Comment:
I think the _MSC_VER >= 1800 is not needed… if _WIN32_WINNT is at least vista all stuff should be available…
Or why do you think the check for compiler version is needed?